src/storm_jsonrpc.js

```diff
@@ -0,0 +1,122 @@
+import fs from 'fs'
+import path from 'path'
+import StormRunner from 'storm'
+
+const RpcServer = require('http-jsonrpc-server');
+import log from './logfile.js'
+
+var testCasePath;
+
+// To get the location of the testcase within testcases folder  which needs to executed
+function getTestcasePath(dirPath, testCase, arrayOfFiles) {
```

Comment:
I wasn't able to actually run a test because the `getTestcasePath`-method doesn't properly resolve to a testcase file.

The Storm-CLI uses a simple method to retrieve all testcases (even nested in subfolders) that you might want to look at https://github.com/WebPlatformForEmbedded/Storm-CLI/blob/master/testcases/index.js

Reply:
Corrected the logic to get into the correct directory. 
Now the getTestcasePathis working fine.